content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/cards.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: cards/cards
+cards:
+  - headline: User research
+    text: Do user research in every iteration of development 
+    link: https://guides.service.gov.au/user-research/
+  - headline: Service design and delivery process
+    text: Own the whole user experience
+    link: https://guides.service.gov.au/service-design-delivery-process/whole-user-experience/
```

Comment:
Please use the relative path instead of the whole URL

Reply:
Steve fixed